css/style.css

```diff
@@ -127,40 +131,36 @@ div.HPS_gform_card_icon {
 }
 
 /*error message*/
-.gform_wrapper .validation_message,
-.gform_wrapper div.validation_error,
-.gfield_error
+li div.hps_secure_ach ~ div.validation_message,
```

Comment:
It looks like it is supported in all the browsers we test for.. am I missing something? http://caniuse.com/#search=~